benchmarks/inference/mii/README.md

```diff
@@ -61,9 +61,35 @@ figures will be saved to `./plots/`
 - `src/plot_repl_scale.py`: This script will plot the throughput and number of replicas for a fixed clients/replica per plot.
 - `src/plot_tp_sizes.py`: This script will plot latency and TFLOPs per GPU across different tensor parallelism sizes.
 
-The following command shows an example of `plot_th_lat.py` execution using the `vllm`, `fastgen`, and `aml` backends.
+## Throughput Latency Plot Generation Script
+The `plot_th_lat.py` throughput-latency plot generation script is generalized for any result output directory, irrespective of where it was run.
+
+The script uses an **_optional_** `plot_config.yaml` that resides within each result directory and allows for overrides in the plot formatting. An example config file may look like this:
+```yaml
+config:
```

Comment:
Is it necessary to have this `config:` inside a `plot_config.yaml` file? Can we remove this?

Reply:
Removed `config:` and updated script.